modules/server/src/test/ts/ClisTest.ts

```diff
@@ -94,11 +95,68 @@ describe('Clis.forAuto', () => {
       bucket: 3,
       buckets: 7,
       useSandboxForHeadless: false,
+      extraBrowserCapabilities: '',
       skipResetMousePosition: false,
       wipeBrowserCache: false
     }, cleanResult(actual));
   });
 
+  it('parses extra browser capabilities arguments with a leading space', () => {
+    const args = [
+      '--browser', 'chrome-headless',
+      '--files', 'src/test/resources/test.file1',
+      '--config', 'src/test/resources/tsconfig.sample.json',
+      '--extraBrowserCapabilities', ' --some-browser-flag'
+    ];
+    const actual = Clis.forAuto(directories, args);
+    AttemptUtils.assertResult({
+      browser: 'chrome-headless',
+      bundler: 'webpack',
+      config: 'src/test/resources/tsconfig.sample.json',
+      name: 'bedrock-run',
+      output: 'scratch',
+      help: false,
+      testfiles: [
+        'src/test/resources/test.file1'
+      ],
+      debuggingPort: 9000,
+      delayExit: false,
+      singleTimeout: 30000,
+      stopOnFailure: false,
+      overallTimeout: 600000,
+      loglevel: 'advanced',
+      version: false,
+      chunk: 100,
+      retries: 0,
+      polyfills: [ 'Symbol' ],
+      bucket: 1,
+      buckets: 1,
+      useSandboxForHeadless: false,
+      extraBrowserCapabilities: ' --some-browser-flag',
+      skipResetMousePosition: false,
+      wipeBrowserCache: false
+    }, cleanResult(actual));
+  });
+
+  // This isn't so much of desired test outcome, just a reality. If we don't have 
```

Comment:
Will it work with quoted strings like `--foo "--bar"` or does it need to be prefixed by a space. That will be very hard to spot.

Reply:
Yeah, it's a compromise. The `command-line-arguments` that bedrock is currently using just uses regex to identify what everything is. Anything that starts with a --, even if it has quotes surrounding it, is considered an option.  The quotes just make it a single value to `argv`. It doesn't need to be a space ... just anything, because the code it is trying to bypass is this: 

```
const re = {
  short: /^-([^\d-])$/,
  long: /^--(\S+)/,
  combinedShort: /^-[^\d-]{2,}$/,
  optEquals: /^(--\S+?)=(.*)/
}
```

Reply:
I think the only real solution is to use yargs, because they've already solved this problem: https://github.com/yargs/yargs-parser/issues/145, but that's a bigger chunk of work.

Reply:
Right that makes sense. I see now that the option description tells you about this requirement.